### PR TITLE
Add secondary language options

### PR DESCRIPTION
**Pull request by gmertes**
The original `settings.html` already had a section for the secondary language options, but it was hidden. Instead of adding a default language option as proposed in #5, the format of this existing menu seemed better. It adds 3 buttons so that the user can choose which behaviour they want: subtitles disabled, use the audio language, or use the last used language.

image

This PR implements the logic behind these options. The 'audio' option is identical to the current behaviour (try to match subtitle language to audio language). For 'last used' , it will save the bcp47 code of the last activated subtitle in the options, and try to activate it when playback is started. Disabled turns off automatically enabling subs when playback is started, but the user can still manually activate one.

This feature introduces a substantial change in how the injected agent 'talks' to the background worker. Currently, a connection to the background worker is made by the render loop on the start of playback (this handles passing `gRenderOptions` and (de)saturating the extension icon). The problem is that this connection is made _after_ playback is started, which means we only get our `gRenderOptions` after playback is started. Since it is passed asynchronously, we also don't know exactly when we will receive it so moving the connection to the start of the render loop doesn't guarantee we will receive the options in time.

TLDR: we do not have access to the language options until after playback is started. We could work around it by activating the subtitles when we receive the options instead of in the render loop, but having the render loop be responsible for connection with the background worker is also dirty and could use a rework anyway. 

So I changed the connection logic so that a connection is made as soon as the injected agent is loaded. That way, we also get an updated copy of `gRenderOptions` immediately from the background worker. I also added 2 new messaging events to handle the (de)saturation of the extension Icon (`startPlayback` and `stopPlayback`), instead of having this happen when a connection is made. Now, connection to the background worker is independent of the render loop which is a lot cleaner imo.

Feedback is welcome. I'm currently still testing to see if there are any side effects to this change, but so far it seems to work fine. 

**Known issue:**
- Desaturation of the icon is currently not triggered when playback is stopped. The message to desaturate the icon (and stop the render loop) is sent in `RendererLoop.stop()` , but the current hook does not trigger it. 
- Since the above hook isn't triggered, the icon will stay saturated after the first playback. 
- The injected agent now stays connected to the background worker for as long as the tab stays open and there is no explicit disconnect. I don't think this is a problem, but feedback is welcome.

### CODE REVIEW

File: src/nflxmultisubs.js [hunk 1]
```diff
@@ -731,6 +768,7 @@ class RendererLoop {
 
     // this script may be loaded while user's at the movie list page,
     // thus if there's no video playing, we can end the renderer loop
+    // FIXME: doesn't trigger when videoplayer is closed
     if (!this.videoElem && !/netflix\.com\/watch/i.test(window.location.href)) {
       this.stop();
```
[the remark this answers is not in the document]
Comment (gmertes) [reply src/nflxmultisubs.js:832]: Fixed in v2.2.5
